koku/reporting/provider/aws/openshift/models.py

```diff
@@ -95,6 +95,10 @@ class Meta:
     cluster_alias = models.CharField(max_length=256, null=True)
     unblended_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     markup_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    blended_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    markup_cost_blended = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    savingsplan_effective_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    markup_cost_savingsplan = models.DecimalField(max_digits=33, decimal_places=15, null=True)
```

Comment:
Can we please standardize the max_digits and decimal_places? Let's use `max_digits=33, decimal_places=15` for all of the new fields.

Reply:
@maskarb thanks for pointing these out